modules/running-compliance-scans-worker-node.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-scans.adoc
+
+[id="running-compliance-scans-worker-node_{context}"]
+=  Scheduling the result server pod on a worker node
+
+The result server pod is used to mount the persistent volume (PV) storage volume that stores the raw Asset Reporting Format (ARF) scan results. The `nodeSelector` and `tolerations` attributes allow you to configure the location of the result server pod.
+
+This is helpful for those environments where control plane nodes are not permitted to mount persistent volumes.
+
+.Procedure
+
+. Inspect the `ScanSetting` object by running:
```

Comment:
One more nit -- this object doesn't exist by default. So we can't tell the user to inspect it before it's created